docs/en/license.rst

```diff
@@ -0,0 +1,29 @@
+License
```

Comment:
Use `.. include:: ../../LICENSE` to avoid duplicate

Reply:
good catch 👍  done.